content/about/about.html

```diff
@@ -15,48 +15,48 @@
   <body>
     <main>
       <h1>About</h1>
+      <!-- 翻訳元リビジョン: https://github.com/w3c/aria-practices/tree/7f54d9620e32231021aad607b7af7eeee5c24f54 -->
       <ul>
+        <li lang="ja">
+          <h2><a href="introduction/introduction.html">はじめに</a></h2>
+          <p>
+            APG の目的、前提知識、およびその構成について学ぶには、 <a href="introduction/introduction.html">はじめに</a> をお読みください。
+          </p>
+        </li>
         <li>
-          <h2><a href="introduction/introduction.html">Introduction</a></h2>
+          <h2><a href="aria-basics/aria-basics.html">ARIA の基本</a></h2>
           <p>
-            To learn  about the purpose of the APG, the prerequisite knowledge it  assumes, and how it is organized, read the
-            <a href="introduction/introduction.html">Introduction</a>.
+            このセクションでは <a href="aria-basics/aria-basics.html">ARIA の基本</a> について説明します。ARIA が何であるか、アクセシビリティのセマンティクスが何であるか、そしてそれらの目的と限界をしっかりと理解することは、APG（Accessible Rich Internet Applications Practices Guide）のガイダンスを適用する際の成功を容易にします。
           </p>
         </li>
         <li>
-          <h2><a href="aria-basics/aria-basics.html">ARIA Basics</a></h2>
+          <h2><a href="contributing/contributing.html">APG への貢献</a></h2>
           <p>
-            Facilitate success when applying APG guidance by developing a solid understanding of what ARIA is, what accessibility semantics are, and  their purpose and limitations with this section about
-            <a href="aria-basics/aria-basics.html">ARIA Basics</a>.
+            APG タスクフォースは、APG の有用性と品質を継続的に向上させるために、広範囲なコミュニティの代表と参加に依存しています。 <a href="contributing/contributing.html">アクセシブルな体験の開発を促進する方法として、あなたがどのように貢献できるかを学びましょう</a> 。
```

Comment:
representationは「意見」とするのが自然かなと思います。また「依存している」より「頼りにしている」くらいの方が堅くなくて良いかなと。
また後半は、learnするのはcontributeする方法で、それによってpromoteをhelpするという構文なのかなと思いました。

```suggestion
            APG タスクフォースは、APG の有用性と品質を継続的に向上させるために、幅広いコミュニティからの意見と参加を頼りにしています。<a href="contributing/contributing.html">あなたがどのように貢献できるかを知り、アクセシブルな体験の開発を促進するためにご協力ください</a> 。
```

ご意見いただきたく！